gsd-schema/osv-gsd/README.md

```diff
@@ -0,0 +1,76 @@
+# OSV GSD Schema
+
+The GSD project primarily relies on the OSV schema for data, but we use an enhanced version of the currently released OSV schema. The reason for this is that the OSV takes time to release new versions of the schema, and in order to prevent blocking the release of data, we use the extended version which is backward compatible with the OSV schema. Using the standard OSV parser will result in warnings (e.g. some ENUM values we use are not present in OSV), but simply ignore the extended fields and attributes you will be able to process the data.
+
+# OSV GSD Extensions
+
+## severity["type": "CVSS_V2"]
+
+Upstream: https://github.com/ossf/osv-schema/pull/87
+Released in OSV schema: 1.4.0
```

Comment:
Anything that gets released in upstream should be removed from the list, imo.

Reply:
No, we show everything that has been accepted, basically new stuff at the bottom, old stuff at the top.